content/blog/2020-08-19-train-a-sequence-model-with-poutyne/index.en.md

```diff
@@ -0,0 +1,613 @@
+---
+title: Training a Recurrent Neural Network (RNN) using PyTorch
+author: David Beauchemin and Marouane Yassine
+date: '2020-08-18'
+slug: machine learning
+type: post
+categories: ["python", "machine learning", "RNN", "PyTorch", "Poutyne", "LSTM", "Address parsing"]
+tags: []
+description: "Train a RNN for address parsing"
+featured: "we-mastered-the-art-of-programming-cover.jpeg"
+featuredpath: "img/headers/"
+---
+
+> In this article, we will train an RNN, or more precisely, an LSTM, to predict the sequence of tags associated with a 
```

Comment:
I think both can be said but "an" seems a bit more popular according to [Google Ngrams](https://books.google.com/ngrams/graph?content=an+RNN%2Ca+RNN%2Ca+LSTM%2C+an+LSTM&year_start=1800&year_end=2019&corpus=26&smoothing=3&direct_url=t1%3B%2Can%20RNN%3B%2Cc0%3B.t1%3B%2Ca%20RNN%3B%2Cc0%3B.t1%3B%2Ca%20LSTM%3B%2Cc0%3B.t1%3B%2Can%20LSTM%3B%2Cc0).

Reply:
I've looked it up recently and I think "an" is the good one. Reading the sentence, we would say RNN (and not Recurrent Neural Network), so that we treat R as if it was written "arr", and so we use "an" because it starts with a vowels. In other words, for acronyms, we should focus on how it sounds and not what the the first letter is.

Reply:
Good enough for me. My interpretation was based on the long version, but this makes much sense.

Btw, didn't know about Google Ngrams, such a nice tool !